cmd/ilm-tier-add.go

```diff
@@ -65,6 +65,21 @@ var adminTierAddFlags = []cli.Flag{
 		Value: "",
 		Usage: "Azure Blob Storage account key",
 	},
+	cli.StringFlag{
+		Name:  "sp-tenant-id",
+		Value: "",
+		Usage: "Directory ID for the service principal account",
+	},
+	cli.StringFlag{
+		Name:  "sp-client-id",
+		Value: "",
+		Usage: "The client ID of the service principal account",
+	},
+	cli.StringFlag{
+		Name:  "sp-client-secret",
+		Value: "",
+		Usage: "The client secret of the service principal account",
+	},
```

Comment:
These all flags need to say that they are for azure so prefix it with `azure-sp-`

Reply:
> These all flags need to say that they are for azure so prefix it with azure-sp-

Done